horovod/tensorflow/mpi_ops.py

```diff
@@ -17,6 +17,7 @@
 # =============================================================================
 """Inter-process communication using MPI."""
 
+import sys
```

Comment:
Nit: try to maintain alphabetical order.  Standard import order is:

```
import system_import
from system_import import something

import third_party_import
from third_party import something

import horovod_imports
from horovod import something
```

Within each of those, we use alphabetical order.